src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_UIElement.TransformToVisual.cs

```diff
@@ -0,0 +1,138 @@
+				},
+				ItemTemplate = new DataTemplate(() => new Border
+				{
+					Width = 200,
+					Height = 100,
+					Background = new SolidColorBrush(Colors.Red),
+					Margin = new Thickness(0, 5),
+					Child = new TextBlock().Apply(tb => tb.SetBinding(TextBlock.TextProperty, new Binding()))
+				}),
+				ItemsSource = Enumerable.Range(1, 10),
+				Margin = new Thickness(15)
+			};
+			var sut = new Grid
+			{
+				Height = 300,
+				Width = 200,
+				Children = { listView }
+			};
+
+			TestServices.WindowHelper.WindowContent = sut;
+			await TestServices.WindowHelper.WaitForIdle();
+
+			AssertItem(0); // Top item, fully visible
+			AssertItem(2); // Bottom item, partially visible
+			// AssertItem(5); // Overflowing item, not materialized => No container for this
+			// AssertItem(9); // Last item, definitely not materialized => No container for this
+
+			void AssertItem(int index)
+			{
+				const double defaultTolerance = 1.5;
+				var tolerance = defaultTolerance * Math.Min(index + 1, 3);
+
+				var container = listView.ContainerFromIndex(index) as ContentControl
+					?? throw new NullReferenceException($"Cannot find the container of item {index}");
+				var border = container.FindFirstChild<Border>()
+					?? throw new NullReferenceException($"Cannot find the materialized border of item {index}");
+
+				var containerToListView = container.TransformToVisual(listView).TransformBounds(new Rect(0, 0, 42, 42));
+				Assert.IsTrue(Math.Abs(containerToListView.X) < tolerance);
+				Assert.IsTrue(Math.Abs(containerToListView.Y - ((100 + 5 * 2) * index)) < tolerance);
+				Assert.IsTrue(Math.Abs(containerToListView.Width - 42) < tolerance);
+				Assert.IsTrue(Math.Abs(containerToListView.Height - 42) < tolerance);
+
+				var borderToListView = border.TransformToVisual(listView).TransformBounds(new Rect(0, 0, 42, 42));
+				Assert.IsTrue(Math.Abs(borderToListView.X) < tolerance);
+				Assert.IsTrue(Math.Abs(borderToListView.Y - ((100 + 5 * 2) * index + 5)) < tolerance);
+				Assert.IsTrue(Math.Abs(borderToListView.Width - 42) < tolerance);
+				Assert.IsTrue(Math.Abs(borderToListView.Height - 42) < tolerance);
+
+				var containerToSut = container.TransformToVisual(sut).TransformBounds(new Rect(0, 0, 42, 42));
+				Assert.IsTrue(Math.Abs(containerToSut.X - 15) < tolerance);
+				Assert.IsTrue(Math.Abs(containerToSut.Y - (15 + (100 + 5 * 2) * index)) < tolerance);
+				Assert.IsTrue(Math.Abs(containerToSut.Width - 42) < tolerance);
+				Assert.IsTrue(Math.Abs(containerToSut.Height - 42) < tolerance);
+
+				var borderToSut = border.TransformToVisual(sut).TransformBounds(new Rect(0, 0, 42, 42));
+				Assert.IsTrue(Math.Abs(borderToSut.X - 15) < tolerance);
+				Assert.IsTrue(Math.Abs(borderToSut.Y - (15 + (100 + 5 * 2) * index + 5)) < tolerance);
+				Assert.IsTrue(Math.Abs(borderToSut.Width - 42) < tolerance);
+				Assert.IsTrue(Math.Abs(borderToSut.Height - 42) < tolerance);
```

Comment:
```suggestion
				using(new AssertionScope())
				{
					borderToSut.Should().Be(
						borderToSut.X - 15,
						borderToSut.Y - (15 + (100 + 5 * 2) * index + 5)),
						borderToSut.Width - 42,
						borderToSut.Height - 42,
						tolerance);
					}
```

... using this: https://github.com/unoplatform/uno/blob/master/src/Uno.UI.RuntimeTests/Helpers/SizeAssertion.cs

Reply:
... and you can wrap into an `using(new AssertionScope())` to let execute all the tests instead of failing on the first one.